i18n/en.json

```diff
@@ -2334,6 +2334,10 @@
     "id": "api.templates.post_body.button",
     "translation": "Go To Post"
   },
+  {
+    "id": "api.templates.remove_expired_license.body.message",
+    "translation": "Remove the expired license in System Console > Edition and License to stop receiving these emails."
```

Comment:
```suggestion
    "translation": "You can remove the expired license in **System Console > Edition and License** to stop receiving these emails."
```

Reply:
We don't support markup in email templates. Trying to bold using html also doesn't work. I'll remove the background and change the text.